site/en/docs/crux/api/index.md

```diff
@@ -145,7 +145,7 @@ These percentiles can show specific metric values at the given percentile for th
 Note: The values for each percentile are synthetically derived, it does not imply that any user actually experienced the value indicated, only that some percentage of users experienced a metric value that was less than the value given.
 {% endAside %}
 
-### Metric value types
+#### Metric value types
```

Comment:
Demoted these so they are under the `Metric` heading. Downside of this, is that they no longer appear in the navigation menu:

<img width="205" alt="image" src="https://user-images.githubusercontent.com/10931297/195068441-e1cdb08b-5660-4f5a-8e97-f2d959b8b96d.png">

WDYT?

Reply:
SGTM